src/groupeddataframe/complextransforms.jl

```diff
@@ -92,9 +92,91 @@ function fill_row!(row, outcols::NTuple{N, AbstractVector},
     return nothing
 end
 
-function _combine_rows_with_first!(first::Union{NamedTuple, DataFrameRow},
+function _combine_rows_with_first_task!(tid::Integer,
+                                        rowstart::Integer,
+                                        rowend::Integer,
+                                        i::Integer,
```

Comment:
maybe use something else than `i` for this? as below `for i in i:rowend` looks a bit unclear.